public/app/core/utils/explore.test.ts

```diff
@@ -113,31 +113,6 @@ describe('state functions', () => {
           '{"expr":"super{foo=\\"x/z\\"}","refId":"B"}],"range":{"from":"now-5h","to":"now"}}'
       );
     });
-
-    // TODO: remove in 9.0
-    it('returns url parameter value for a state object', () => {
-      const state = {
-        ...DEFAULT_EXPLORE_STATE,
-        datasource: 'foo',
-        queries: [
-          {
-            expr: 'metric{test="a/b"}',
-            refId: 'A',
-          },
-          {
-            expr: 'super{foo="x/z"}',
-            refId: 'B',
-          },
-        ],
-        range: {
-          from: 'now-5h',
-          to: 'now',
-        },
-      };
-      expect(serializeStateToUrlParam(state, true)).toBe(
-        '{"datasource":"foo","queries":[{"expr":"metric{test=\\"a/b\\"}","refId":"A"},{"expr":"super{foo=\\"x/z\\"}","refId":"B"}],"range":{"from":"now-5h","to":"now"}}'
-      );
```

Comment:
as far as i can tell, these don't test anything that the other tests aren't already testing so i think they can just be removed 😄

Reply:
@Elfo404 This is what you were saying about testing the deprecated `compact` parameter - right?

Reply:
yeah, basically my reasoning for not removing them back then was to make sure the `compact` parameter was not going to assume a different meaning before we actually removed it. but honestly i agree with @ashharrison90 they are practically useless